hypha/apply/api/v1/views.py

```diff
@@ -41,7 +41,7 @@
 )
 
 
-class SubmissionViewSet(viewsets.ReadOnlyModelViewSet):
+class SubmissionViewSet(viewsets.ModelViewSet):
```

Comment:
ModelViewSet comes with lot of inbuilt features(it even allows us to delete objects, basically whole CRUD.)

I think it is needed for the `set_summary` put action..?

We can use [GenericViewSet](https://www.django-rest-framework.org/api-guide/viewsets/#genericviewset) for that along with ReadOnlyModelViewSet:

```
class SubmissionViewSet(viewsets.ReadOnlyModelViewSet, viewsets.GenericViewSet):
```